verification/verifier_helpers.go

```diff
@@ -253,12 +253,12 @@ func (v *Verifier) executePlugin(ctx context.Context, trustPolicy *TrustPolicy,
 	if err != nil {
 		return nil, err
 	}
-	var attributesToProcess []interface{}
-	extendedAttributes := make(map[interface{}]interface{})
+	var attributesToProcess []string
+	extendedAttributes := make(map[string]interface{})
 
 	for _, attr := range getNonPluginExtendedCriticalAttributes(signerInfo) {
-		extendedAttributes[attr.Key] = attr.Value
-		attributesToProcess = append(attributesToProcess, attr.Key)
+		extendedAttributes[attr.Key.(string)] = attr.Value
```

Comment:
This line of code would panic if attr.Key is not of type string. Type assertion is needed here.

Reply:
This is handled in `getNonPluginExtendedCriticalAttributes`. It is returning only the string attributes